src/openforms/emails/utils.py

```diff
@@ -100,3 +103,23 @@ def render_email_template(
         backend=openforms_backend,
         disable_autoescape=disable_autoescape,
     )
+
+
+def collect_failed_emails(since: datetime) -> list[dict[str, str] | None]:
+    logs = TimelineLogProxy.objects.filter(
+        timestamp__gt=since,
+        extra_data__status=Message.STATUS_FAILED,
+        extra_data__include_in_daily_digest=True,
+    ).distinct("content_type", "extra_data__status", "extra_data__event")
+
+    if not logs:
+        return
```

Comment:
Probably you can return an empty list anyway, makes the return type a bit simpler?

Reply:
+1 from me on this